libs/src/sdk/api/tracks/types.ts

```diff
@@ -114,7 +129,7 @@ export const createUploadTrackSchema = () =>
     .object({
       userId: HashId,
       coverArtFile: ImageFile,
-      metadata: createUploadTrackMetadataSchema(),
+      metadata: createUploadTrackMetadataSchema().strict(),
```

Comment:
Maybe we put this `strict()` on the schema itself? That way we don't need to do it in both upload and update

Reply:
is there a way to unstrict a schema? needed it to allow for other keys in the upload flow

Reply:
This allows unrecognized keys: https://github.com/colinhacks/zod#passthrough

But yeah if we need it in upload flow then let's leave it how you have it

Reply:
i dont think we need to pass the other fields through, we just need to not throw an error when there are keys outside of the schema.
and yea ok, going to leave as is for now and fix if there are problems

Reply:
Ah ok then `.strip()` is what we would want